src/v1/vouchers.ts

```diff
@@ -0,0 +1,89 @@
+import { Client } from '../client'
+import { BaseError } from '../errors'
+import { UriHelper } from '../uri-helper'
+import * as v0 from '../v0'
```

Comment:
This import is way too big for what is necessary here, but probably doesn't matter as it's cached.

Still, why not only: `import { Vouchers as VouchersV0, VouchersOptions } from from '../v0/vouchers'`

Reply:
That's definitely a good point. removed import all

Reply:
:ok_hand: